src/propeller/core.cljc

```diff
@@ -3,18 +3,26 @@
   (:require [propeller.gp :as gp]
             [propeller.problems.simple-regression :as regression]
             [propeller.problems.string-classification :as string-classif]
-            [propeller.problems.software.number-io :as number-io]
-            [propeller.problems.software.smallest :as smallest]
             #?(:cljs [cljs.reader :refer [read-string]])))
 
+(defn eval-problem-var
+  [problem-name var-name]
+  (eval (symbol (str "propeller.problems." problem-name "/" var-name))))
```

Comment:
Here we assume that all problems are under `propeller.problems.`. Does this seem reasonable or is this too restrictive?

(We lifted most of this logic from Clojush.)